src/components/portal/portal.tsx

```diff
@@ -101,6 +102,7 @@ export class Portal {
         const content = slot.assignedElements();
 
         this.container = document.createElement('div');
+        this.container.style.opacity = '0';
```

Comment:
Super minor, but since we have a `showContainer`, we could have a `hideContainer` as well :smile:

Reply:
Yea thats true! I will change that :)